.asf.yaml

```diff
@@ -20,6 +20,12 @@
 github:
   description: "Apache Pulsar - distributed pub-sub messaging system"
   homepage: https://pulsar.apache.org/
+  notifications:
+    commits:      commits@pulsar.apache.org
+    issues:       commits@pulsar.apache.org
+    pullrequests: commits@pulsar.apache.org
+    discussions:  dev@pulsar.apache.org
```

Comment:
It would be good to do this consistently across all the repositories

Reply:
Also, discussion are not enabled yet, was there already consensus to enable it?

Reply:
I think so, ML: https://lists.apache.org/thread/1y7zbchlbokwnpd0jv2tt5jtzg6px6yn

Reply:
@merlimat @Anonymitaet I don't think it's an explicit consensus.

For @Anonymitaet's comment below. Yes all events under discussion will be sent to the mailing list. I've asked for separating messages by categorizes (https://github.com/apache/infrastructure-github-discussions-notifier/commit/09b7206dc6b4f6ad970954d9482beade62cb3400#r76983377) but it seems no response yet. Perhaps you can create an INFRA ticket for it.

Reply:
> It would be good to do this consistently across all the repositories

@merlimat Totally agree. Actually the main repo is set to send all these activities (expect discussion since it's not turned on) to commits@. The GitHub activities sent to dev@ nosily are almost sent from other pulsar repos. This is pointed out by @dave2wave at https://lists.apache.org/thread/hy2yj5l636zctnb5631ypmp2bs8flthc.

Reply:
Regarding the response from INFRA:

> Please confirm that the Project is aware and agrees that the use of Discussions are for user-facing topics only,

I think we may redirect discussions to users@ and narrow the discussion categorizes to Q&A and Announcement.

I will send a formal VOTE thread on the mailing list to see if there is a consensus.

Reply:
Discussions is now enabled